modules/aleph/src/Aleph/AlephDebtMaterial.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+/**
+ * Class AlephDebtMaterial.
+ *
+ * Material information related to debt.
```

Comment:
There's year and author, however it's used nowhere yet.
Should we add it anyway, @danquah?